### PR TITLE
Increase font size and change default fonts.

### LINKED ISSUE #609: Increase font size, improve readability and make a better use of font sizes in general.
Currently A+ use a `font-size: 14px` and it makes it difficult to read the text in small devices. Of course, we can make use of the zoom but increasing the base font might be beneficial. 
https://css-tricks.com/accessible-font-sizing-explained/

Depends on #594

### PR DESCRIPTION
**Pull request by jaguarfi**
# Description

Ths PR intends to improve the accessibility of A+ by increasing the font size from 14px to 16px, which is the standard size in
many browsers. This PR also change the default fonts for optimum text rendering on every device and OS.

The changes are implemented by overwriting the Bootstrap variables. This will give the opportunity to remove or edit these changes whenever it is considered necessary (Perhaps when we start using a newer version of Bootstrap)

Fixes #609

# Testing

The changes in the font size and font families should not affect the behaviour of the A+ course. However, I manually check that every page of the Aplus Manual was not affected by the changes.

# Have you updated the README or other relevant documentation?

There is no need for updating the documentation

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

*Clean up your git commit history before submitting the pull request!*

- [ ] I (developer) have created unit tests and the tests pass
- [ ] I (developer) have created functional tests (Selenium tests) if applicable
- [X] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

File: assets/sass/main.scss [hunk 1]
```diff
@@ -18,6 +18,14 @@ $brand-info: $aplus-info;
 $brand-warning: $aplus-warning;
 $brand-danger: $aplus-danger;
 
+// import new font variables
+@import 'variables/typography';
+$font-family-sans-serif: $aplus-font-family-sans-serif;
+$font-family-serif:$aplus-font-family-serif;
```
Comment (Mankro) [inline assets/sass/main.scss:24]: Whitespace missing after the colon.

File: assets/sass/main.scss [hunk 2]
```diff
@@ -18,6 +18,14 @@ $brand-info: $aplus-info;
 $brand-warning: $aplus-warning;
 $brand-danger: $aplus-danger;
 
+// import new font variables
```
Comment (Mankro) [inline assets/sass/main.scss:21]: The other comments have been started with a capital letter so this one could too.

**Review by Mankro — CHANGES_REQUESTED** [review]
Great! Just need to minify the output CSS.

File: assets/css/main.css [hunk 3]
```diff
@@ -1,9 +1,8368 @@
-﻿/*!
+@charset "UTF-8";
```
[the remark this answers is not in the document]
Comment (jaguarfi) [reply assets/css/main.css]: I forgot to run that command. Now it should be ok.

**Review by Mankro — APPROVED** [review]

**Pull request merged**